src/content/community/Community-Users.md

```diff
@@ -0,0 +1,121 @@
+---
+title: GraphQL for Users
+layout: docs
+category: Community
+permalink: /community/users/
+next: /community/developers/
+sublinks: Training courses,Where to ask questions,Blogs,Videos,Books,More resources
+---
+
+The following resources are intended to help users and implementers. Find something useful? Please add it to the list!
+
+*Please note that the GraphQL Foundation does not provide support or services, and cannot recommend one vendor's product over another.*
+
+## Training courses
+
+A number of GraphQL training courses are available.
+
+* [Exploring GraphQL: A Query Language For APIs](https://www.edx.org/course/exploring-graphql-a-query-language-for-apis): A free 7 week edX course, developed by [Linux Foundation Training](https://training.linuxfoundation.org)
+* [How to GraphQL](https://www.howtographql.com): The Fullstack Tutorial for GraphQL
+
+## Where to ask questions
+
+### Stack Overflow
+
+Many members of the community use Stack Overflow to ask and answer questions. If you're running into an implementation issue or having trouble with a query, this is the best place to start.
+
+[Read through the existing questions tagged with **graphql** or ask your own!](http://stackoverflow.com/questions/tagged/graphql)
+
+### Slack, Discord, and IRC
+
+Many GraphQL developers idle in Discord, Slack, and IRC for live
+communication and can answer quick questions. 
+
+Please be patient and polite. These are not explicitly user support channels, although they're likely a good place to find someone who can answer your question if you ask nicely.
+
+* **Discord:** [#graphql on the ReactiFlux Discord](http://join.reactiflux.com/)
+* **Slack:** [#general on the GraphQL Slack](https://graphql.slack.com/messages/general/) [(Get an invite)](https://graphql-slack.herokuapp.com/)
+* **IRC:** [#graphql on freenode](https://freenode.net/)
+
+### Vendor channels
+
+The following channels are maintained by vendors, and may be a good place to find help on their products.
+
+* [Apollo Slack](http://apollostack.com/#slack)
+* [Prisma Slack](https://slack.prisma.io/)
```

Comment:
Not related and has nothing to do with the review so if it's a problem we'll add it later but maybe you could add also The Guild's Discord channel here?

https://discord.gg/xud7bH9

Also worth mentioning that the link to Discord in the section above makes it look like Reactiflux is the official GraphQL Discord channel which is really not, it's just a small section on their Discord

Reply:
Yeah, I wondered what the criteria for getting listed here is! If you're adding community chats, it's https://discord.gg/graphile for the Graphile Discord :+1:

Reply:
Yes, let's add both! The main thing is that I'd like to ensure we're clear which is a general community channel, and which are run by organizations in the ecosystem. Just so there's no confusion.

Reply:
I think in terms of Discord, there is no general community, so maybe put all Discord channels together under the Vendor section?
For me the Reactiflux one is also a "Vendor" but I might be wrong so maybe someone else can confirm